Source/CarthageKit/Dependency.swift

```diff
@@ -100,12 +109,29 @@ extension Dependency: Hashable {
 		case let .git(url):
 			return url.hashValue
 
-		case let .binary(url):
-			return url.hashValue
+		case let .binary(binary):
+			return binary.hashValue
 		}
 	}
 }
 
+extension Dependency.Binary: Comparable {
+	public static func == (_ lhs: Dependency.Binary, _ rhs: Dependency.Binary) -> Bool {
+		return lhs.absoluteURL == rhs.absoluteURL &&
+				lhs.representation == rhs.representation
+	}
+
+	public static func < (_ lhs: Dependency.Binary, _ rhs: Dependency.Binary) -> Bool {
```

Comment:
Why not just Equatable?

Reply:
You're right, I misread the Dependency comparable as a requirement for this implementation. Will mark just equatable.